package.json

```diff
@@ -16,7 +16,8 @@
     "dev": "snowpack dev --config snowpack.dev.config.cjs",
     "start": "yarn dev",
     "test": "web-test-runner --coverage --config test/web-test-runner.config.js",
-    "publish-patch": "yarn run build && np patch --no-tests --any-branch"
+    "publish-patch": "yarn run build && np patch --no-tests --any-branch",
+    "postinstall": "mkdirp sandbox"
```

Comment:
I'm a little unclear why this is necessary. If all we need to do is add an unignored file like in video.js, we should just do that. Or maybe it's pushing the sanbox dir to the repo without it being ignored first.

Reply:
not at all necessary if we do what you proposed. I just already added this. Happy to back out and do what you proposed.